scripts/build-shotcut.sh

```diff
@@ -44,7 +44,6 @@ LIBOPUS_HEAD=1
 LIBOPUS_REVISION=
```

Comment:
In my opinion, all you would need to do to accomplish your goal would be to preload the existing "XXX_REVISION" variables. Everything else in this script could stay the same.